RecoBTag/ONNXRuntime/interface/tensor_configs.h

```diff
@@ -28,4 +30,85 @@ namespace deepvertex {
 
 }  // namespace deepvertex
 
+namespace parT {
+
+  enum InputFeatures {
+    kBegin=0,
+    kChargedCandidates=kBegin,
+    kNeutralCandidates=1,
+    kVertices=2,
+    kChargedCandidates4Vec=3,
+    kNeutralCandidates4Vec=4,
+    kVertices4Vec=5,
+    kEnd=6
+  };
+
+  constexpr unsigned n_cpf_accept = 25;
+  constexpr unsigned n_npf_accept = 25;
+  constexpr unsigned n_sv_accept = 5;
+
+  const std::map<InputFeatures, unsigned int> N_InputFeatures{
```

Comment:
this should be `nInputFeatures` to follow the CMS naming rules: http://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1
(similarly for the other similar variables below)